collect_app/src/main/res/layout/waveform_layout.xml

```diff
@@ -5,10 +5,9 @@
     android:layout_height="match_parent"
     app:chunkAlignTo="bottom"
     app:chunkColor="?colorSecondary"
-    app:chunkMaxHeight="80dp"
```

Comment:
Weird to duplicate this value here as it meant changing the height in the parent had no effect. The value is now set in `onLayout()`.